cfme/tests/configure/test_log_depot_operation.py

```diff
@@ -311,26 +244,57 @@ def _clear_ftp():
         if len(datetimes) > 1:
             for i in range(len(datetimes) - 1):
                 dt = datetimes[i + 1][0] - datetimes[i][1]
-                soft_assert(
-                    dt.total_seconds() >= 0.0,
+                assert dt.total_seconds() >= 0.0, \
                     "Negative gap between log files ({}, {})".format(
-                        datetimes[i][2], datetimes[i + 1][2]))
+                        datetimes[i][2], datetimes[i + 1][2])
 
 
 @pytest.mark.tier(3)
-def test_collect_unconfigured(request, soft_assert):
+@pytest.mark.nondestructive
+@pytest.mark.meta(blockers=[BZ(1341502, unblock=lambda depot_type: depot_type != "anon_ftp",
+                            forced_streams=["5.6", "5.7", "5.8", "upstream"]),
+                            BZ(1436367, forced_streams=["5.8"])]
+                  )
+def test_collect_log_depot(depot_type, depot_machine_folder, depot_credentials, depot_ftp,
+                           configured_depot, depot_machine_ip,
+                           request):
+    """ Boilerplate test to verify functionality of this concept
+
+    Will be extended and improved.
+    """
+    # Wipe the FTP contents in the end
+    @request.addfinalizer
+    def _clear_ftp():
+        with depot_ftp(depot_machine_ip) as ftp:
+            ftp.cwd(ftp.upload_dir)
+            ftp.recursively_delete()
+
+    # Prepare empty workspace
+    with depot_ftp(depot_machine_ip) as ftp:
+        # move to upload folder
+        ftp.cwd(ftp.upload_dir)
+        # delete all files
+        ftp.recursively_delete()
+
+    # Start the collection
+    configured_depot.collect_all()
+    # Check it on FTP
+    check_ftp(depot_ftp(depot_machine_ip))
+
+
+@pytest.mark.meta(blockers=[BZ(1436367, forced_streams=["5.8"])])
+@pytest.mark.tier(3)
+def test_collect_unconfigured(soft_assert):
     """ Test checking is collect button enable and disable after log depot was configured
 
     """
-    request.addfinalizer(configure.ServerLogDepot.Credentials.clear)
-    log_credentials = configure.ServerLogDepot.Credentials("smb",
-                                                           "testname",
-                                                           "testhost",
-                                                           username="testusername",
-                                                           password="testpassword")
-    log_credentials.update(validate=False)
+    log_credentials = configure.ServerLogDepot("anon_ftp",
+                                               depot_name=fauxfactory.gen_alphanumeric(),
+                                               uri=fauxfactory.gen_alphanumeric())
+
+    log_credentials.create()
     # check button is enable after adding log depot
-    soft_assert(toolbar.is_greyed("Collect", "Collect all logs") is False)
-    configure.ServerLogDepot.Credentials.clear()
+    assert toolbar.is_greyed("Collect", "Collect all logs") is False
```

Comment:
**Rerquired**
consider navigating first so you can be assured you're on the right page. 

Model the toolbar as a nested view so this can be done using widgetastic instead of relying on web_ui.toolbar.